core/controllers/load_functions.py

```diff
@@ -117,7 +112,9 @@ def delete_existing_submission(submission_to_del: str) -> None:
     db.session.flush()
 
 
-def get_or_generate_submission_id(workbook: dict[str, pd.DataFrame]) -> tuple[str, Submission | None]:
+def get_or_generate_submission_id(
+    programme_exists_same_round: Programme, reporting_round: int
+) -> tuple[str, Submission | None]:
```

Comment:
Docstring description needs updating to reflect these changes. It still references "worksheet" which was a param before the func was refactored